client/my-sites/domains/domain-management/name-servers/wpcom-nameservers-toggle.jsx

```diff
@@ -1,15 +1,17 @@
-import { Card } from '@automattic/components';
+import { Card, ScreenReaderText } from '@automattic/components';
 import { ToggleControl } from '@wordpress/components';
 import { localize } from 'i18n-calypso';
 import PropTypes from 'prop-types';
 import { PureComponent } from 'react';
 import { connect } from 'react-redux';
+import FormLabel from 'calypso/components/forms/form-label';
 import { CHANGE_NAME_SERVERS } from 'calypso/lib/url/support';
 import {
 	composeAnalytics,
 	recordGoogleEvent,
 	recordTracksEvent,
 } from 'calypso/state/analytics/actions';
+const NAMESERVERS_TOGGLE_TITLE = 'Use WordPress.com Name Servers';
```

Comment:
Nice job moving the label to a `const`, but I think they wouldn't be translated like this. 
You also need to add the `__` module (`@wordpress/i18n` package) call to retrieve the translation of the text. 

[Here](https://developer.wordpress.org/reference/functions/__/) are the docs for it. 

There's also a similar example here:

https://github.com/Automattic/wp-calypso/blob/f8c7fe5df453bde9a7582df5c092821baf3ef6a0/client/components/domains/connect-domain-step/constants.ts#L55-L68

Reply:
Moving from a localized and translated string to non-translated one is definitely a regression as @rafaelgalani recognized.

Furthermore, I don't think this is an instance that requires a getter function like in the constants file in the previous comment. For me, the problem wasn't that we're duplicating the same text twice; but rather that we need to have a separate `span` with a visual label, and then the toggle separated semantically from it, while the label and the toggle are disconnected from each other.

Reply:
>Furthermore, I don't think this is an instance that requires a getter function like in the constants file in the previous comment.

@tyxla If the const was declared inside the class (constructor, method, etc), then you're right; but if the constant was kept outside React's context, then I think this would be necessary since the translations wouldn't be retrieved as the HOC props were updated.

Reply:
@rafaelgalani see https://github.com/Automattic/wp-calypso/pull/57374#discussion_r743654922 - as part of my review I'm suggesting an alternative solution that keeps the string inside the React component, which is why I've suggested that the getters would be unnecessary overkill.